src/extensions/default/BrambleUrlCodeHints/camera/index.js

```diff
@@ -8,11 +8,11 @@ define(function (require, exports, module) {
 
     var CommandManager = brackets.getModule("command/CommandManager");
     var Commands       = brackets.getModule("command/Commands");
+    var fs             = brackets.getModule("fileSystemImpl");
```

Comment:
Curious why you chose to use this level of abstraction for the filesystem?  We don't do caching at this level.  Is this why you're doing the `FileSystemCache.refresh`?